packages/ui/src/hooks/useStore.ts

```diff
@@ -102,10 +102,10 @@ export const useStore = (): Store => {
       confirmJobActions
     );
 
-  const retryAll = (queueName: string) =>
+  const retryAll = (queueName: string, status: JobFinishedStatus) =>
     withConfirmAndUpdate(
-      () => api.retryAll(queueName),
-      'Are you sure that you want to retry all jobs?',
+      () => api.retryAll(queueName, status),
+      'Are you sure that you want to retry all failed jobs?',
```

Comment:
The message is not modified by the status, it can be `failed` or `completed`.

Reply:
That's why I initially implemented it using two separate functions. `cleanAll` already has 3 variants for `failed`, `completed`, and `delayed` so I assumed this approach would be fine for retries as well